### ISSUE TITLE
Docker image does not build

### ISSUE THREAD

**Issue by mrhanlon** [TACC/agavepy#7]
I am getting the error:

```
Step 5 : RUN /opt/conda/envs/python2/bin/python setup.py develop
 ---> Running in 5d5c92e16232
running develop
running egg_info
creating agavepy.egg-info
error: could not create 'agavepy.egg-info': Permission denied
ERROR: Service 'agavepy' failed to build: The command '/bin/sh -c /opt/conda/envs/python2/bin/python setup.py develop' returned a non-zero code: 1
```

**Comment by waltermoreira** [comment]
Yep, confirmed.  It seems that the base jupyter image changed its structure of users.

**Comment by waltermoreira** [comment]
Fixed in commit 7c1e8d434978a26c153bb090e44d3616f574a836
